tests/testthat/test-utils_data.R

```diff
@@ -0,0 +1,50 @@
+test_that("rownames_as_column works", {
+  test <- rownames_as_column(mtcars, "new_column")
+  expect_true("new_column" %in% names(test))
+  expect_true(test[1, "new_column"] == "Mazda RX4")
+})
+
+test_that("rownames_as_column doesn't work if var is not a character", {
+  expect_error(rownames_as_column(mtcars, var = 1))
```

Comment:
If you want, you can also add an expected error message to `regexp` argument in `expect_error()`, so that we know that the functions are not working for expected reasons.

Reply:
Good idea! I didn't know there was a regexp argument